view/theme/frio/img/dynamic.php

```diff
@@ -7,7 +7,7 @@
       'dsn' => 'mysql:host=127.0.0.1;dbname=installer',
       'username' => 'root',
       'password' => 'root',
-      'charset' => 'utf8',
+      'charset' => 'utf8mb4',
```

Comment:
I recognized it yesterday for the first time while reviewing this PR. This file seems to be a mistake. It can be deleted.